docs/Users_Guide/release-notes.rst

```diff
@@ -43,6 +43,56 @@ When applicable, release notes are followed by the
 describes the bugfix, enhancement, or new feature.
 Important issues are listed **in bold** for emphasis.
 
+METplus Version 6.1.0 Beta 2 Release Notes (2025-04-01)
+-------------------------------------------------------
+
+  .. dropdown:: Enhancements
+
+     * **Update Python from 3.10.x to 3.12.0**
+       (`#2697 <https://github.com/dtcenter/METplus/issues/2697>`_)
+     * Update the EnsembleStat_fcstICAP_obsMODIS_aod use case to create meaningful output
+       (`#2791 <https://github.com/dtcenter/METplus/issues/2791>`_)
+     * **Install METplus via Conda**
+       (`#2863 <https://github.com/dtcenter/METplus/issues/2863>`_)
+     * Support time templates in field name for SeriesAnalysis
+       (`#2880 <https://github.com/dtcenter/METplus/issues/2880>`_)
+     * Resolve findings from SonarQube
+       (`#2923 <https://github.com/dtcenter/METplus/issues/2923>`_)
+     * Update FV3 Physics Tendency use case data
+       (`#2937 <https://github.com/dtcenter/METplus/issues/2937>`_)
+     * PairStat - Add support for time filtering options
+       (`#2944 <https://github.com/dtcenter/METplus/issues/2944>`_)
+
+  .. dropdown:: Bugfix
+
+     NONE
+
+  .. dropdown:: New Wrappers
+
+     NONE
+
+  .. dropdown:: New Use Cases
+
+     * GFS cloud forecasts vs. GFS cloud analyses
+       (`#2743 <https://github.com/dtcenter/METplus/issues/2743>`_)
+     * GFS cloud forecasts vs. GOES-16 cloud products
+       (`#2744 <https://github.com/dtcenter/METplus/issues/2744>`_)
+     * GFS cloud forecasts vs. ASOS ceiolometer
+       (`#2745 <https://github.com/dtcenter/METplus/issues/2745>`_)
+
+  .. dropdown:: Documentation
+
+     * Update existing use cases to use the template
+       (`#2741 <https://github.com/dtcenter/METplus/issues/2741>`_)
+     * Use subprojects in Read The Docs
+       (`#2771 <https://github.com/dtcenter/METplus/issues/2771>`_)
+
+  .. dropdown:: Build, repository, and test
+
+     * Update the Existing Builds page and modulefiles for the 6.0.0 coordinated release
```

Comment:
```suggestion
     * Update the Existing Builds page and modulefiles for the 6.0.0/6.1.0 coordinated release
```
I just changed the title of the issue to add 6.1.0, which is what went into the develop branch.  The 6.0.0 updates went into the main branch.  Sorry for the confusion.